stisim/celery.py

```diff
@@ -0,0 +1,82 @@
+# FUNCTIONS TO RUN MODELS ON CLUSTER
```

Comment:
I wonder whether this file should live here or somewhere else, e.g. the analyses folder... @alina-muellenmeister @RomeshA how have you done this with other work?

Reply:
We had it together with the interventions and networks modules for the gavi work, but I am not sure either if it should live somewhere else

Reply:
It would normally be in the analyses folder - there is some commonality across analyses but separating that out seems like more trouble than it would be worth